### ISSUE TITLE
Minor error in nn.MultiMarginLoss documentation

### ISSUE THREAD

**Issue by Hv0nnus** [pytorch/pytorch#7237]
Hello,

In torch.nn.MultiMarginLoss in the Pytorch documentation : https://pytorch.org/docs/stable/nn.html?highlight=multimarginloss#torch.nn.MultiMarginLoss.

The loss in the case where there is weights is : 
\[\text{loss}(x, y) = \frac{\sum_i \max(0, w[y] * (\text{margin} - x[y] - x[i]))^p)}{\text{x.size}(0)}\]
and should be
\[\text{loss}(x, y) = \frac{\sum_i \max(0, w[y] * (\text{margin} - x[y] + x[i]))^p)}{\text{x.size}(0)}\]
The + has change to a - between the loss without weight and with weight , which is probably not what is coded.
